kge/model/embedder/lookup_embedder.py

```diff
@@ -84,11 +83,20 @@ def _embed(self, embeddings: Tensor) -> Tensor:
         return embeddings
 
     def embed(self, indexes: Tensor) -> Tensor:
-        return self._embed(self.embeddings(indexes.long()))
+        if indexes is None:
+            return self._embed(self._get_all_parameters())
+        else:
+            return self._embed(self.embeddings(indexes.long()))
 
     def embed_all(self) -> Tensor:
-        self.all_indexes = self.all_indexes.to(self.embeddings.weight.device)
-        return self.embed(self.all_indexes)
+        return self.embed(None)
+
+    def _get_all_parameters(self) -> Tensor:
```

Comment:
Names are quite confusing (already before this PR). I suggest:
- Rename `self.embeddings` to `self._embeddings`
- Rename this function to `self._embeddings_all` (for consistency with embed and embed_all)
- Rename `self._embed` to `self._postprocess`